### ISSUE TITLE
Build a web interface to display CSJ data from webservice in tabular form

### ISSUE THREAD

**Issue by artitangri** [codeforsanjose/Project-Ideas#86]
City of San Jose has a web service that returns various datasets. City needs help with building a web UI to display the data from web service on to a web page in a tabular form. 
An example of data - http://api.sanjoseca.gov/OpenDataService.svc/PlanningPermitsDatas?$format=json&$filter=Status%20eq%20%2730%27 
this is all the planning permits issued by the city in last 30 days.

The goal is to have a generic solution that can be easily and quickly implemented for new datasets.

**Comment by thielCole** [comment]
@3vivekb Is this webpage going to be connected to a different project or will it be siloed in itself?

**Comment by 3vivekb** [comment]
@thielCole This project will be contained within the ckan portal for the vta/san jose collaboration.

It would be a data preview in a site like http://data2.vta.org/dataset/test_082516_01.

@Kyle-Falconer could speak about this with more detail.

**Comment by artitangri** [comment]
Yes, this is for city of san jose to be able to publish live data from API to the CKAN portal. I will be there tomorrow to answer any questions.

**Comment by thielCole** [comment]
So you just want a tab style box that allows you to export the data to csv?

**Comment by thielCole** [comment]
@artitangri Do you know what the values mean? If you did I could make some visuals for it. 

Here are the values

              "Status":"30",
        "gx_location":"  CITYWIDE   , SAN JOSE CA ",
        "ASSESSORS_PARCEL_NUMBER":null,
        "ZONING":"A",
        "APPLICANT":"Building DIvision, PBCE, City of San Jose Mr James Son",
        "OWNERNAME":"CITY OF SAN JOSE  City Attorney ",
        "ISSUEUSER":"TCHAULE",
        "PROJECTMANAGER":null,
        "FOLDERNUMBER":"2016-132501-PP",
        "FOLDERDESCRIPTION":"Envir Public Projects",
        "FOLDERNAME":"PP16-090",
        "REFERENCEFILE":"PP16-090",
        "SUBDESCRIPTION":"Exempt",
        "WORKDESCRIPTION":null,
        "ISSUEDATE":"2016-09-02T00:00:00",
        "FINALDATE":null,
        "FOLDERRSN":"1684603"

**Comment by thielCole** [comment]
Another set of examples incase it helps.

         "Status":"30",
         "gx_location":"1057  LINCOLN AV  , SAN JOSE CA 95125-6011",
         "ASSESSORS_PARCEL_NUMBER":"26457074",
         "ZONING":"CP",
         "APPLICANT":" JASON LAMMERS",
         "OWNERNAME":" JASON LAMMERS",
         "ISSUEUSER":"DFONG",
         "PROJECTMANAGER":"DFONG",
         "FOLDERNUMBER":"2016-105616-DV",
         "FOLDERDESCRIPTION":"Development Permit",
         "FOLDERNAME":"SP16-013",
         "REFERENCEFILE":"SP16-013",
         "SUBDESCRIPTION":"Special Use Permit",
         "WORKDESCRIPTION":"None",
         "ISSUEDATE":"2016-09-21T00:00:00",
         "FINALDATE":"2016-10-03T00:00:00",
         "FOLDERRSN":"1657572"

**Comment by artitangri** [comment]
This is how we would like to have it look like (in a tabular format)
http://data.sanjoseca.gov/dataviews/230615/LAST-30-DAYS-PLANN-PERMI/ 

Ideally, the solution should be generic that it can be adapted to any dataset. So basically, depending on what data the API returns it should pick up the column names and display the data in a tabular format. No hard-coded column names.

**Comment by Kyle-Falconer** [comment]
Here's the repo for this project: https://github.com/codeforsanjose/odata-reader
